.github/workflows/release.yaml

```diff
@@ -17,29 +17,22 @@ jobs:
     secrets: inherit
 
   release:
-    runs-on: ${{ matrix.runs-on }}
+    runs-on: ubuntu-22.04
     needs: check
     strategy:
       fail-fast: false
       matrix:
-        runs-on: [[ubuntu-22.04], [self-hosted, jammy, ARM64]]
-    outputs:
-      snap: ${{ steps.build.outputs.snap }}
+        archs: ["aarch64", "x86_64"]
```

Comment:
Hmm how does this interact with other release.yaml templates? Because this still requires two jobs to run.  I was hoping there'd be some way to autodetect the artifacts and download and publish them in a loop in the same job. :thinking:

Reply:
I didn't fully understand your question.

The other snap releases would need to populate the necessary `archs`  and I think it would be fine for just amd64 projects

Reply:
I mean this refactoring doesn't improve efficiency, so probably better (and less work on changing the templates) to leave it as `runs-on: [[ubuntu-22.04], [self-hosted, jammy, ARM64]]`.

If we have a method that reduces this from two job runs to one, then it's worth it.

Just my 2c.

Reply:
The advantage would be running without the need of an arm runner, but I guess in the end that would introduce a new variable that it's not used in other workflows.

In any case I've changed back to use the runner